### PR TITLE
add anger questions for quiz

### PR DESCRIPTION
**Pull request by nferner**
Proposed 5 questions/scenarios for crank-o-meter

### CODE REVIEW

File: ReadME.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+##**Anger Questions**
+Please use the following questions to rank your crank on a scale of 1-3 (3 being extremely cranky)
```
Comment (arianacofone) [inline ReadME.md]: Should we include what 1 would represent in the range?

File: ReadME.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+1. How cranky does being stuck behind an old couple slowly walking hand in hand on a small sidewalk on a busy street make you?
+2. How cranky do you get when you're out to eat and no one has come over to take your order and the only way to get anyone's attention is to actually get up and introduce yourself as a patron?
+3. How does it make you feel when you're at the gym and a dude is sitting at the machine you need and is just texting?
```
Comment (arianacofone) [inline ReadME.md]: Can we use "someone" instead of dude to make it more gender neutral?

**Review by arianacofone — COMMENTED** [review]
This made me laugh pretty intensely. Nice job! Before approving, can we try to make the sentences more concise?

File: ReadME.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-Please use the following questions to rank your crank on a scale of 1-3 where 1 is annoyed and 3 is raging.
-1. How cranky does being stuck behind an old couple slowly walking hand in hand on a small sidewalk on a busy street make you?
-2. How cranky do you get when you're out to eat and no one has come over to take your order and the only way to get anyone's attention is to actually get up and introduce yourself as a patron?
-3. How does it make you feel when you're at the gym and someone is sitting at the machine you need and is just texting?
+Please rank your crank based on the scenarios below where 1 is annoyed and 3 is raging.
+1. You're stuck behind an old couple slowly walking hand in hand on a small sidewalk on a busy street. How cranky does this make you?
+2. You're out to dinner and no one has come to take your order. They are actually so far away from you that you would have to get up an introduce yourself to the waiter.'How cranky does this make you?
+3. You're working out at the gym and someone is sitting at the machine you need and just texting. What's your crank rank?
 4. How angry were you when Geoffrey put Ned's head on spike and made Sansa look at it?
```
Comment (arianacofone) [inline ReadME.md]: I love this one, but maybe we should make it more neutral for none Game of Throne fans...bummer, I know. What do you think?

**Review by arianacofone — APPROVED** [review]
Great changes Neil!

File: ReadME.md (hunk 1 above)
Comment (tbongiorno1) [reply ReadME.md]: Second Ariana's point

File: ReadME.md (hunk 2 above)
Comment (tbongiorno1) [reply ReadME.md]: Some of the questions seem a bit run on, maybe rethink some wording?

**Pull request merged**